### ISSUE TITLE
[TW-1037] due.before:eow not working

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1072]
_Paul Beckingham on 2010-06-25T08:43:37Z says:_ 

 Michelle says:

(As an aside, I notice that my recurring task, which is due on Friday (tomorrow right now), refuses to show up when I ask for tasks due before eow. Is there something about recurring tasks that breaks the deadlines?)

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-06-25T08:43:37Z
Modified: 2014-02-09T02:15:15Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-06-27T14:31:01Z says:_ 

 Probably already fixed when #416 was fixed.
